servers/src/main/java/tachyon/master/file/meta/Inode.java

```diff
@@ -278,6 +295,54 @@ public synchronized void setPinned(boolean pinned) {
     mPinned = pinned;
   }
 
+  /**
+   * @return the username of the inode
+   */
+  public synchronized String getUserName() {
+    return mUserName;
+  }
+
+  /**
+   * Sets the userName of the inode.
```

Comment:
the method-level comment for setter is not required as it does not add new information